Source/WebCore/Modules/websockets/WorkerThreadableWebSocketChannel.h

```diff
@@ -33,6 +33,7 @@
 #include "ThreadableWebSocketChannel.h"
 #include "WebSocketChannelClient.h"
 #include "WorkerGlobalScope.h"
+#include "WorkerLoaderProxy.h"
```

Comment:
Why is this needed? I only WorkerLoaderProxy being used a reference in a class definition here.